### ISSUE TITLE
Export of photos library with folders > 10 only gets first 10 directories

### ISSUE THREAD

**Issue by sa2000a** [ukdtom/plex2csv.bundle#91]
Does not appear to recurse to do next 10 directories 
A photos library with over 200 directories only has first 10 directories processed

```
2015-11-18 21:41:55,900 (46cc) :  DEBUG (base:117) - Checking if com.plexapp.plugins.Plex2csv is broken
2015-11-18 21:41:55,900 (46cc) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X1N0b3JlU2VydmljZTpJc0NoYW5uZWxCcm9rZW4_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMQpzMjgKY29tLnBsZXhhcHAucGx1Z2lucy5QbGV4MmNzdnMxMAppZGVudGlmaWVycjAK'
2015-11-18 21:41:55,931 (46cc) :  DEBUG (runtime:88) - Sending packed state data (109 bytes)
2015-11-18 21:41:55,931 (46cc) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 3617 bytes
2015-11-18 21:41:56,121 (4d88) :  DEBUG (runtime:717) - Handling request GET /applications/Plex2csv/backgroundScan?title=Photos&random=1716.67634552&key=8&sectiontype=photo
2015-11-18 21:41:56,122 (4d88) :  DEBUG (runtime:814) - Found route matching /applications/Plex2csv/backgroundScan
2015-11-18 21:41:56,122 (4d88) :  DEBUG (logkit:13) - ******* Starting backgroundScan *********
2015-11-18 21:41:56,122 (4d88) :  DEBUG (runtime:1117) - Created a thread named 'backgroundScanThread'


2015-11-18 21:41:56,124 (3ec0) :  DEBUG (logkit:13) - *******  Starting backgroundScanThread  ***********
2015-11-18 21:41:56,124 (3ec0) :  DEBUG (logkit:13) - Section type is photo
2015-11-18 21:41:56,124 (3ec0) :  DEBUG (logkit:13) - Path to medias in selection is http://127.0.0.1:32400/library/sections/8/all
2015-11-18 21:41:56,125 (3ec0) :  DEBUG (logkit:13) - Output file is named D:\Plex_PMS_LocalData\Plex Media Server\plex2csv-exports\Plex2csv\Photos-Level 3-20151118-214156.csv
2015-11-18 21:41:56,125 (3ec0) :  DEBUG (logkit:13) - ******* Starting scanPhotoDB with an URL of http://127.0.0.1:32400/library/sections/8/all ***********
2015-11-18 21:41:56,125 (3ec0) :  DEBUG (logkit:13) - Writing headers for Photo Export
2015-11-18 21:41:56,125 (3ec0) :  DEBUG (logkit:13) - Starting to fetch the list of items in this section
2015-11-18 21:41:56,125 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/sections/8/all?type=10&X-Plex-Container-Start=0&X-Plex-Container-Size=0'
2015-11-18 21:41:56,164 (3ec0) :  DEBUG (logkit:13) - Walking medias
2015-11-18 21:41:56,164 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/sections/8/all?X-Plex-Container-Start=0&X-Plex-Container-Size=10'
2015-11-18 21:41:56,188 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18623/children?includeExtras=1'
2015-11-18 21:41:56,813 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18776/children?includeExtras=1'
2015-11-18 21:41:56,884 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18791/children?includeExtras=1'
2015-11-18 21:41:56,905 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18795/children?includeExtras=1'
2015-11-18 21:41:56,950 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18804/children?includeExtras=1'
2015-11-18 21:41:57,084 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18837/children?includeExtras=1'
2015-11-18 21:41:57,098 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18839/children?includeExtras=1'
2015-11-18 21:41:57,142 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18848/children?includeExtras=1'
2015-11-18 21:41:57,193 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18857/children?includeExtras=1'
2015-11-18 21:41:57,233 (3ec0) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/18865/children?includeExtras=1'
2015-11-18 21:41:57,296 (3ec0) :  DEBUG (logkit:13) - ******* Ending scanPhotoDB ***********
2015-11-18 21:41:57,296 (3ec0) :  DEBUG (logkit:13) - *******  Ending backgroundScanThread  ***********
2015-11-18 21:41:58,124 (4d88) :  DEBUG (logkit:13) - ************** Scan Done, stopping wait **************
2015-11-18 21:41:58,124 (4d88) :  DEBUG (logkit:13) - *******  All done, tell my Master  ***********
2015-11-18 21:41:58,124 (4d88) :  DEBUG (logkit:13) - *******  Ending complete  ***********
2015-11-18 21:41:58,125 (4d88) :  DEBUG (runtime:88) - Sending packed state data (109 bytes)
2015-11-18 21:41:58,125 (4d88) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 702 bytes
2015-11-18 21:42:05,588 (4c08) :  DEBUG (runtime:717) - Handling request GET /applications/Plex2csv
2015-11-18 21:42:05,589 (4c08) :  DEBUG (runtime:814) - Found route matching /applications/Plex2csv
```

**Comment by ukdtom** [comment]
Hopefully fixed with the newest version on the tommy-work branch

The darn counter on the top is however still an issue, but please take it out for a spin

**Comment by ukdtom** [comment]
And forgot.....Bumped up the container size to 20

**Comment by ukdtom** [comment]
Fixed in 0.0.4.2
